src/app/containers/MediaPlayer/Metadata.jsx

```diff
@@ -14,4 +15,32 @@ const Metadata = aresMediaBlock => {
   );
 };
 
+Metadata.propTypes = {
+  aresMediaBlock: shape({
+    model: shape({
+      blocks: arrayOf(
+        // aresMediaMetadata
+        shape({
+          model: shape({
+            format: string,
+            title: string,
+            synopses: shape({
+              short: string,
+            }),
+            versions: arrayOf(
+              shape({
+                duration: number,
+                availableFrom: number,
+              }),
+            ),
+            imageUrl: string,
+          }),
+        }),
+        // image
+        shape({}),
```

Comment:
I don't believe the above will work as expected. Let's remove the `blocks` definition and just make the `model` property required.

Reply:
It's worth noting the metadata helper, which this data is passed to, does a fairly good job of dealing with the variability of the blocks. I agree with @jamesdonoh that this component should just be responsible for ensuring the helper gets that data.